src/main/msp/msp.c

```diff
@@ -1893,11 +1893,12 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
 #if defined(USE_INTERPOLATED_SP)
         sbufWriteU8(dst, currentPidProfile->ff_interpolate_sp);
         sbufWriteU8(dst, currentPidProfile->ff_smooth_factor);
+        sbufWriteU8(dst, currentPidProfile->ff_boost);
```

Comment:
This should come later, once the format for this has been solidified by being exposed to all users.

Reply:
OK, removed for later